sydent/http/servlets/store_invite_servlet.py

```diff
@@ -102,6 +105,9 @@ def render_POST(self, request):
         if substitutions["room_name"] != '':
             substitutions["bracketed_room_name"] = "(%s)" % substitutions["room_name"]
 
+        if 'web_client_location' not in substitutions:
+            substitutions["web_client_location"] = 'https://app.element.io'
```

Comment:
It'd be nice to have this as a config option instead... but this could be easily done later on if needed.

Reply:
I think this was done in bab81965b04300b736d1bf19263a4617723bb2da.